packages/api/src/schema.ts

```diff
@@ -1617,6 +1713,8 @@ const schema = gql`
     generateApiKey(input: GenerateApiKeyInput!): GenerateApiKeyResult!
     unsubscribe(name: String!): UnsubscribeResult!
     subscribe(name: String!): SubscribeResult!
+    setWebhook(input: SetWebhookInput!): SetWebhookResult!
```

Comment:
Does `setWebhook` create a new one?

Reply:
It can be used to create a new one or update an existing one